go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -119,32 +119,37 @@ func TestSingleScript(t *testing.T) {
 
 	var scripts = []queries.ScriptTest{
 		{
-			Name: "insert duplicate key doesn't prevent other updates, autocommit off",
```

Comment:
Do you mean to be entirely replacing the "insert duplicate key doesn't prevent other updates, autocommit off" test?

Reply:
This is just a convenience test method to debug a single script at a time. There's a skip at the top of it.